general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -213,7 +227,49 @@ def get_job(self, job_id: str) -> Dict[str, str]:
         Raises:
             SuperstaqServerException: For other API call failures.
         """
-        return self.get_request(f"/job/{job_id}")
+        warnings.warn(
+            "Starting with client-superstaq v0.4.20, the `get_job` method and "
+            "endpoint won't exist. Call `fetch_jobs` instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+        return self.fetch_jobs([job_id])[0]
+
+    def fetch_jobs(
+        self,
+        job_ids: List[str],
+        tags: Optional[List[str]] = None,
+        targets: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> List[Dict[str, str]]:
+        """Get the job from the Superstaq API.
+
+        Args:
+            job_ids: The UUID of the job (returned when the job was created).
+            tags: optional tags of jobs
+            targets: optional targets of jobs
+            kwargs:  Extra options needed to fetch jobs.
+                - cq_token: CQ Cloud credentials.
+
+        Returns:
+            The json body of the response as a dict.
+
+        Raises:
+            SuperstaqServerException: For other API call failures.
+        """
+
+        json_dict: Dict[str, Any] = {
+            "job_ids": job_ids,
+        }
+
+        if tags:
+            json_dict["tags"] = tags
+        if targets:
+            json_dict["targets"] = targets
+        if kwargs or self._client_kwargs:
+            json_dict["options"] = json.dumps({**kwargs, **self._client_kwargs})
```

Comment:
```suggestion
            json_dict["options"] = json.dumps({**self._client_kwargs, **kwargs})
```
(so passed-in arguments take precedence)

Reply:
Done.